### PR TITLE
530 index

### PR DESCRIPTION
**Pull request by akonsowski**
this fix is to make the security marking when submitting a listing clearer to the user. please advise if i should change the wording to listing instead of application.  note  this fix requires pull request [#56](https://github.com/ozone-development/ozp-react-commons/pull/56) before working

### CODE REVIEW

File: app/js/components/createEdit/index.jsx [hunk 1]
```diff
@@ -344,7 +344,7 @@ var ListingForm = React.createClass({
             <form ref="form" className="CreateEdit__form col-xs-9 col-lg-10">
                 <h2 id={f.basicInformation.id}>Basic Information</h2>
 
-                <MarkingInput id={f.securityMarking.id} { ...p('securityMarking') } aria-label="Classification and Control Marking"/>
+                <MarkingInput id={f.securityMarking.id} { ...p('basicSecurityMarking') } aria-label="Classification and Control Marking for the Application"/>
```
Comment (wski) [inline app/js/components/createEdit/index.jsx]: What does ...p('basicSecurityMarking') return? We should just get the text for aria-label that way. I realize you didn't make the 'aria-label' but we should change how we pull the text for it if we're going to modify it anyway.

**Comment by akonsowski** [comment]
@SOAP1914 can you check the aria-labels and make sure they are alight with the screen reader. i checked with the console log but i just want to make sure

**Comment by SOAP1914** [comment]
👍

**Comment by wski** [comment]
👍

**Pull request merged**